src/main/java/school/lemon/changerequest/java/introduction/hw2/Task10.java

```diff
@@ -2,6 +2,18 @@
 
 public class Task10 {
     public static String trim(String text) {
-        return "";
+
+        if(text == null)
```

Comment:
As it's said in the task definition:
```
 Implement your own solution based on Regexp.
```

Please adjust your solution and use Regular Expressions.